### ISSUE TITLE
Consider revising the names of some metrics

### ISSUE THREAD

**Issue by andrewhowdencom** [bakins/php-fpm-exporter#1]
Recently, i submitted this exporter to the list of exporters for Prometheus in https://github.com/prometheus/docs/pull/791. Brian Brazil appears to have reviewed the upstream application itself, and given some suggested for amended metric names:

```
Counters should end in _total

listen_queue_length and max_listen_queue are inconsistent, listen_queue_length and listen_queue_max would be better.
```

Would you consider amending the names of these metrics? I understand this is a breaking change, but perhaps it's ealy enough in this projects life that such changes are still possible without much inconvenience. 

I am happy to try and submit a PR to this effect, but I do little golang.

**Comment by andrewhowdencom** [comment]
Update: I will submit a PR if I get time, with a super opinionated set of metrics. I'll describe and justify all changes, and reference them with supporting docs, and keep them separate so the commits can be dropped if required.

**Comment by bakins** [comment]
@andrewhowdencom thanks for the interest. I have no strong feelings about the names. As is, they are just named after the exposed php-fpm name.

Happy to accept a PR as long as we don't drop any of the metrics.

**Comment by andrewhowdencom** [comment]
doneee check pr. <3

**Comment by andrewhowdencom** [comment]
Resolved in #2. That was my first golang contrib! (I think). Haha it makes me happy
